docker/sandbox/Dockerfile

```diff
@@ -70,6 +70,9 @@ COPY --from=base_ /flyteorg/ /flyteorg/
 # Copy entrypoints
 COPY docker/sandbox/flyte-entrypoint-dind.sh /flyteorg/bin/flyte-entrypoint.sh
 
+# Copy kustomization template 
```

Comment:
I would actually prefer to move this to install using helm as we discussed... and maybe enable users to provide a values file to be overridden... basically reducing the places we use kustomize and making sure helm consistently work... I think introducing a kustomization.yaml here makes that job harder....